syntax = "proto3";
package ndn_message;

message PrefixesInStorage {
    message Prefix {
        string name = 1;
    }
    repeated Prefix prefixes = 1;
}

message CommandsInStorage {
    message Command {
        bytes interest = 1;
        int32 seq = 2;
    }
    repeated Command commands = 1;
}